catalyst-gateway/tests/api_tests/integration/signed_doc/__init__.py

```diff
@@ -0,0 +1,204 @@
+import pytest
+from utils import signed_doc
+from typing import Dict, Any, List
+import copy
+import os
+import json
+from api.v1 import document
+from utils import signed_doc, uuid_v7
+from utils.rbac_chain import rbac_chain_factory, RoleID
+
+
+class SignedDocument:
```

Comment:
Can we move everything that is not a test in the corresponding utils file? ie the SignedDocument class and the fixures